install_environment.sh

```diff
@@ -4,6 +4,9 @@ set -ex
 
 apt-get update
 
+# get arguments as an array
```

Comment:
Please write comments starting from upper case letters to be consistent (the same relates to all similar changes in PR)